diagnostic.md

```diff
@@ -5,27 +5,39 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components. Explain why each piece might be it's own component.
 
     ```md
-    <!-- your response here -->
+    Hear me out, let's say we are making a list app. A list has a title and
+    items in the list. Both the title and the list could be components. So, when
+    we are looking at the differesnt lists, each title is rendered as a component
+    and when you click on them, it renders that list. Each item of that list is
+    also a component.
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are created and/or edited to produce a component, and what are their responsibilities?
 
     ```md
-    <!-- your response here -->
+    `component.js` and  `template.hbs` are rendered. Template handles what the user
+    does. So in our hypothetical list app, template handles the data from the model
```

Comment:
I would say the template *displays* the data from the model; the component manages that data.

.5/1